dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/utils/FlinkArgsUtils.java

```diff
@@ -124,6 +124,10 @@ private FlinkArgsUtils() {
 
         ResourceInfo mainJar = param.getMainJar();
         if (mainJar != null) {
+            if (ProgramType.PYTHON == programType) {
+                // -py
+                args.add(Constants.FLINK_CLI_OPTION_PYTHON);
+            }
```

Comment:
These lines should not place here. It's better to modify:
```java
        if (programType != null) {
            if (programType == ProgramType.PYTHON) {
                args.add(Constants.FLINK_PYTHON);        // -py
            } else if (programType != ProgramType.PYTHON && StringUtils.isNotEmpty(mainClass)) {
                args.add(Constants.FLINK_MAIN_CLASS);    //-c
                args.add(param.getMainClass());          //main class
            }
        }
```

Reply:
> These lines should not place here. It's better to modify them to:
> 
> ```java
>         if (programType != null) {
>             if (programType == ProgramType.PYTHON) {
>                 args.add(Constants.FLINK_PYTHON);        //-py
>             } else if (programType != ProgramType.PYTHON && StringUtils.isNotEmpty(mainClass)) {
>                 args.add(Constants.FLINK_MAIN_CLASS);    //-c
>                 args.add(param.getMainClass());          //main class
>             }
>         }
> ```

Hello, I think the `programType != ProgramType.PYTHON` in the else-if brackets is not necessary. Because once this sentence is judged, this item must be true.